src/backend/joanie/core/serializers.py

```diff
@@ -157,7 +358,7 @@ class AddressSerializer(serializers.ModelSerializer):
     Address model serializer
     """
 
-    id = serializers.CharField(source="uid", required=False)
+    id = serializers.CharField(source="uid", required=False, read_only=True)
```

Comment:
Good catch and we should add a test on each API endpoint to secure that id is readonly!?